algebird-test/src/test/scala/com/twitter/algebird/BatchedTest.scala

```diff
@@ -43,6 +44,10 @@ class BatchedLaws extends CheckProperties with Matchers with PropertyChecks {
   testBatchedMonoid[BigInt]("BigInt", 10)
   testBatchedMonoid[BigInt]("BigInt", 100)
   testBatchedMonoid[BigInt]("BigInt", 1000000)
+  testBatchedMonoid[BigDecimal]("BigDecimal", 1)(arbReasonableBigDecimals, implicitly[Monoid[BigDecimal]])
+  testBatchedMonoid[BigDecimal]("BigDecimal", 10)(arbReasonableBigDecimals, implicitly[Monoid[BigDecimal]])
+  testBatchedMonoid[BigDecimal]("BigDecimal", 100)(arbReasonableBigDecimals, implicitly[Monoid[BigDecimal]])
+  testBatchedMonoid[BigDecimal]("BigDecimal", 1000000)(arbReasonableBigDecimals, implicitly[Monoid[BigDecimal]])
```

Comment:
I'm not really happy with the syntax here (the "implicit" on line 30, and maybe the whole of line 30, is spurious). Not quite sure how to do it better. I tried to hide the low-priority implicits brought in by the Arbitrary object, but didn't quite succeed.